src/omni-router/omni-router.ts

```diff
@@ -0,0 +1,106 @@
+import { BigNumber, BigNumberish, ContractTransaction } from 'ethers'
+
+import { Account } from '../account'
+import { Contract } from '../contract'
+import {
+  OmniRouter as OmniRouterInterface,
+  OmniRouter__factory,
+} from '../typechain'
+import { PromiseOrValue } from '../typechain/common'
+import { Address, ChainId, ChainIds } from '../types'
+
+/** ### Default OmniRouter address for network */
+export const defaultOmniRouterAddress: Record<ChainId, Address> = {
+  /** ### Default OmniRouter address on goerli rollup testnet. */
+  [ChainIds.GoerliRollupTestnet]: '0x5888abe26e0b22fede3988f1b463625752245553',
+}
+
+export class OmniRouter extends Contract<OmniRouterInterface> {
+  /**
+   * ### Creates a new OmniRouter instance
+   *
+   * @param account Account to use for signing
+   * @param contract Contract instance or address of the OmniRouterInterface contract
+   *
+   * @returns New OmniRouterInterface token instance
+   */
+  constructor(account: Account, contract: OmniRouterInterface | Address) {
+    super(account, contract, OmniRouter__factory)
+  }
+
+  /**
+   * ### Returns gas limit with 5% buffer
+   * @param estimatedGas
+   * @returns
+   */
+  private _getGasLimit(estimatedGas: BigNumberish): BigNumberish {
```

Comment:
add an optional param instead of hardcoded 95

scale to 10000 BP, e.g `.mul(10000).div(9500)`